.eslintrc.json

```diff
@@ -12,7 +12,7 @@
     "ecmaVersion": 2018,
     "sourceType": "module"
   },
-  "extends": ["airbnb", "plugin:react/recommended", "plugin:react-hooks/recommended"],
```

Comment:
- [ ] Please do not alter any thing in the linter file, please use as provided in the project requirement.